src/views/Conversion.vue

```diff
@@ -247,6 +253,50 @@ export default class Conversion extends Mixins(
     } as EntitySnapshotIF
   }
 
+  /** Fetches the business snapshot for conversion. */
+  private async fetchConversionSnapshot (): Promise<any> {
+    const items = await Promise.all([
+      this.fetchBusinessInfo(),
+      AuthServices.fetchAuthInfo(this.getBusinessId),
+      this.fetchOrgPersons(OrgPersonTypes.PARTIES)
+    ])
+    if (items.length !== 3) throw new Error('Failed to fetch entity snapshot')
+
+    const orgPersons = items[2].map(orgPerson => {
+      const middleInitial = orgPerson.officer['middleInitial']
+      if (middleInitial !== undefined) {
+        orgPerson.officer.middleName = middleInitial
+        delete orgPerson.officer['middleInitial']
+      }
+      return orgPerson
+    })
+
+    return {
+      businessInfo: items[0],
+      authInfo: items[1],
+      orgPersons
+    }
+  }
+
+  private async fetchStoreConversionAddress ():Promise<any> {
+    let hasBAError = false
+    const response = await Promise.resolve(
+      this.fetchConversionAddresses()
+    ).catch(error => {
+      if (error.response.status === 404 &&
+        error.response.config.url.includes('addresses') &&
```

Comment:
This is probably redundant because we know (*) that we called the addresses endpoint.

(*) And we shouldn't need to know that anyway, since this layer of code just deals with conversion data. It's the lower layer that deals with whatever the URL is. The principle here is separation of concerns.

Reply:
There are two types of 404. One with `FM... addresses not found` and one with `FM.... not found`. I had to make sure that this is for `FM... addresses not found` with `addresses` in the error message by including `addresses`.

Reply:
![image](https://user-images.githubusercontent.com/51458187/170362048-9d1bc1cb-2d15-437f-8b4f-854973b87aef.png)

I need to change `addresses` to `address`.

Reply:
OK, thanks for the info. I agree that you should keep the error response message check in here.

Reply:
Please add a code comment explaining why it's necessary to check the message string. (So a future developer -- or me, after I forget -- will understand.)

Reply:
Alright. I added the comment and included "address" differentiate invalid business address from other 404 status.